### ISSUE TITLE
400+ pages book. Does Baker suitable for it?

### ISSUE THREAD

**Issue by divlv** [Simbul/baker#1028]
Hi All!

I need to publish (and sell) the 400+ pages book.
I'm wondering - if Baker the good option for this task?

It is going to be static pages with images,
just like a book for eReaders,
most likely HTMLs with text (not big PNGs) etc.

Anybody got any experience / best practices?

I think, Baker itself should handle this without any problems,
but what about publishing?
Standalone or Newsstand?
I heard something regarding iBooks... but have no much information about it.

Please, share some knowledge.

**Comment by JosephMacReady** [comment]
Hey, take a look at this before considering Baker further: 
http://www.apple.com/ibooks-author/

**Comment by divlv** [comment]
I saw this.
If I understand correctly, it needs separate (paid) Apple Developer account...

**Comment by JosephMacReady** [comment]
Any App Store app needs it, you could start learning iOS development hard or hire a developer if going the app-route. Please read the iBooks Author documentation

**Comment by divlv** [comment]
You didn't understand me correctly, or I have described my problem not clear enough:
I'm talking about completely separate account - " **Paid Books Account** "
mentioned here:
https://itunesconnect.apple.com/WebObjects/iTunesConnect.woa/wo/0.0.0.7.3.0.9.3.1.11.1.0.21

If my current active iOS developer's account (99$/year) is enough for that, or I should
buy one more special account?

That's why I'm asking: 
if somebody got any experience in publishing big books with Baker?

**Comment by folletto** [comment]
I can talk from a technical standpoint: the way Baker manages pages doesn't make any difference between 3 and 400 pages. 

I would suggest to create a "dummy" book with 400 pages and roughly the same number of images you expect to have and see how it behaves and how big the HPub zip file comes out. :)
In this way you'll be sure first-hand about it with a low effort involved. :)

**Comment by rlarrett** [comment]
If you want to sell books (or "books") through iBooks, which is what the "Paid Books Account" refers to, you will need an iTunes login, but you do not need a Developers account. However, iBooks accepts epub or iba (iBooks Author) files, not hpub.  Anything generated via Baker is hpub and by definition an app, and cannot sold through iBooks as "book." Of course there are plenty of book-style apps. If you want to offer epub and also apps, you must have a separate Apple ID account for each, which I discovered the hard way.  Perhaps this is what you mean?

**Comment by divlv** [comment]
Thanks, @rlarrett its definitely closer to what I mean...
So, its not a problem for me (even easier!) to create a 400 pages book like a standalone application with Baker.
Actually, I **want use Baker** for that.
But, it this case, the question may be the following: will Apple accept such kind of book? 
Or, they will tell me  - I need to create iBook to sell such content?
Will it be rejected?

Actually, I heard _somewhere_ that such content (400 pages book like a standalone app) probably is going to be rejected by Apple, until it has some "app-like features" like social networks "buttons" to rate something, or... say... functionality to send e-mail or... something like that.
Is there any info available regarding all this?

**Comment by folletto** [comment]
That's correct. If it's "just a book" Apple is likely to reject it and asking you to publish on the Book Store (regardless if epub or ibook). Baker allows technically any kind of publication, but due to rejection issues you can either go for a book with enhanced interactivity or a Newsstand magazine (being a Newsstand Magazine doesn't require added interactivity, even if it's always a nice to have). :)

**Comment by divlv** [comment]
Aha... actually it's an **ideal variant** for me - to have a book, or, in future, 
even a set of books (say, from one author) **in the Newsstand** shelf.

So, if I'll create my big 400 pages book like a Newsstand application (shelf for the author with a single book so far),
does it have a good chance to be approved by Apple? 
_(subscription feature will be enabled of course)_

**Comment by rlarrett** [comment]
But why risk Apple rejecting your app, when you can sell it as an epub through the iBook Store, without having to pay the Developer account fee? Of course, all ebooks sold in iBooks require an ISBN number, which aren't cheap either. So maybe you're right!

**Comment by folletto** [comment]
Newsstand requirement is being a magazine, or in other words, having a specific periodicity (weekly, monthly, etc). So if you try to build a non-periodic on Newsstand it will be rejected.

**Comment by nfeiglin** [comment]
Interestingly @folletto , my magazine isn't on a such strict schedule and Apple still let it through.

**Comment by folletto** [comment]
Yet another confirmation on how arbitrary are Apple's Policies. :)
